sample_sensor_kit_launch/launch/lidar.launch.xml

```diff
@@ -75,6 +75,22 @@
       </include>
     </group>
 
+    <!-- <group>
```

Comment:
Should not this be deleted?

Reply:
I think that instead of in-code examples, we are focusing on tutorials like this:
https://autowarefoundation.github.io/autoware-documentation/main/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-model/creating-sensor-model/

Reply:
@knzo25 It has been removed, I will create a separate PR for the documentation :+1: